backend/src/handlers/file_map_operations.py

```
@@ -243,22 +223,18 @@ def delete_file_map_handler(event: Dict[str, Any], user_id: str) -> Dict[str, An
-        
-        # if 'Item' not in response:
-        if not file_map:
-            return handle_error(404, f"File map {file_map_id} not found")
-            
-        # file_map = FileMap.model_validate(response['Item'])
-        
-        # Check ownership
-        if str(file_map.user_id) != user_id:
-            return handle_error(403, "Not authorized to delete this file map")
-            
+        file_map = checked_mandatory_file_map(file_map_id, user_id)
```

Ruff (0.12.2)

Local variable `file_map` is assigned to but never used

Remove assignment to unused variable `file_map`

(F841)